### PR TITLE
improving docs for deployment to staging only, and adding index titles

### PR DESCRIPTION
**Pull request by choldgraf**
This adds docs for how to deploy *just* to staging

### CODE REVIEW

**Comment by choldgraf** [comment]
cc @yuvipanda , lemme know how this looks

File: docs/source/deployment/how.md [hunk 1]
```diff
@@ -98,10 +100,37 @@ master.
...
 6. Troubleshoot and make changes to your fork. Repeat the process from Step 1.
 
+### Deploying to *only* `staging`
+
+```eval_rst
+.. note::
+   Currently you cannot deploy changes to `mybinder/requirements.yaml` only to staging.
+```
+Sometimes you want to test out a deployment live before you make a deployment
+to `prod`. This is possible by editing a `staging`-only helm chart. To deploy
```
Comment (yuvipanda) [inline docs/source/deployment/how.md]: staging-only config files.

**Comment by yuvipanda** [comment]
minor fixes, I can correct and merge in a few hours!

**Comment by choldgraf** [comment]
minor fix fixed!

**Comment by yuvipanda** [comment]
thanks @choldgraf!

**Comment by choldgraf** [comment]
🎉

**Pull request merged**